web-components/src/components/menu-overlay/MenuOverlay.ts

```diff
@@ -148,6 +148,8 @@ export namespace MenuOverlay {
 
     connectedCallback() {
       super.connectedCallback();
+      this.handleIframeClick = this.handleIframeClick.bind(this);
```

Comment:
Needed to explicitly bind this here as the context of 'this' would otherwise be lost when iframe is clicked.